modules/sb-start-tsb.adoc

```diff
@@ -16,20 +16,38 @@ After you have installed the {tsb-name} Operator, you can start the
 .Procedure
 
 . Navigate in the web console to *Operators* -> *Installed Operators* and select the `openshift-template-service-broker` project.
+
 . Select the *Template Service Broker Operator*.
-. Under *Provided APIs*, click *Create New* for *Template Service Broker*.
+
+. Under *Provided APIs*, click *Create Instance* for *Template Service Broker*.
+
 . Review the default YAML and click *Create*.
-. Verify that the {tsb-name} has started.
+
+. Verify that the {tsb-name} starts correctly by checking that the template
+applications are available.
 +
-After the {tsb-name} has started, you can view the available
-template applications by navigating to *Catalog* -> *Developer Catalog* and
-selecting the *Service Class* checkbox. Note that it may take a few minutes for
-the {tsb-name} to start and the template applications to be
-available.
+--
+* To check from the web console, switch to the *Developer* perspective, then
+navigate to the *Add+* -> *From Catalog* page. The template applications show up
+as *Template* Type objects.
```

Comment:
Does "Type" need to be capitalized here?

Reply:
Waffled on this. It's capitalized in the web console. But I didn't want to mark it up also. I'll just lowercase it, it's clear enough.